microsoft-edge/webview2/releasenotes.md

```diff
@@ -12,29 +12,68 @@ keywords:  IWebView2, IWebView2WebView, webview2, webview, win32 apps, win32, ed
 
 # Release notes for WebView2 SDK  
 
-The WebView2 team is delivering updates to the [WebView2 SDK][NuGetGallery] on a 6-week cadence.  Review the following content for up to date information about product announcements, additions and modifications to the API surface, and breaking changes.  
+The WebView2 team is delivering updates to the [WebView2 SDK][NuGetGallery] on a six week cadence.  Review the following content for up-to-date information about product announcements, additions and modifications to the API surface, and breaking changes.  
 
 > [!NOTE]
 > Re-compile your app after updating the NuGet package.  
 
 > [!IMPORTANT]
 > While WebView2 is a preview, the .NET APIs are in the `pre-release package`.  
 
+
+## 0.9.628-prerelease  
+Release Date: Sept 09, 2020
+
+[NuGet package][NuGetGallery0.9.579] \| minimum Microsoft Edge version 86.0.579.0.  
```

Comment:
Please update release date as it is postponed until today, add two nuget package entries, and confirm the minimum supported version.